website/docs/tutorials/basic/running_your_app/4_logging.md

```diff
@@ -53,11 +53,13 @@ $ python my_app.py hydra.verbose=[__main__,hydra]
 [2019-09-29 13:06:00,896][__main__][DEBUG] - Debug level message
 ```
 
-You can disable the logging output using by setting `hydra/job_logging` to `disabled'   
+You can disable the logging output by setting `hydra/job_logging` to `disabled`   
 ```commandline
 $ python my_app.py hydra/job_logging=disabled
 <NO OUTPUT>
 ```
 
-Logging can be [customized](/configure_hydra/logging.md).
+You can also set `hydra/job_logging=none` and `hydra/hydra_logging=none` if you do not want Hydra to configure the logging.
+and `hydra/hydra_logging` to `none`
```

Comment:
redundant line?

Reply:
The line you removed was there before I added anything to the documentation, and I think it does serve a purpose there. On another note, I did find that the documentation in general was a bit confusing since certain topics were described multiple times in multiple different places. Perhaps the documentation should be changed in general?

Reply:
I was referring to this part: 
```
Hydra to configure the logging.
and `hydra/hydra_logging` to `none`
```